README.md

```diff
@@ -142,3 +145,16 @@ The following users have deployed JanusGraph in production.
 * [Red Hat](https://www.redhat.com/) - [application migration and assessment tool](https://developers.redhat.com/products/rhamt/overview/) built on [Windup](https://github.com/windup/windup)
 * [Times Internet](https://timesinternet.in)
 * [Uber](https://uber.com)
+
+## Cite
+
+```
+@software{janusgraph_2024,
+    title = {JanusGraph: an open-source, distributed graph database},
+    author = {{JanusGraph Team}},
```

Comment:
Who is _JanusGraph Team_ / who belongs to it?

I think _JanusGraph contributors_ would be better as that is a defined term and should include everyone who ever contributed to JanusGraph, including people who contributed to Titan.

_JanusGraph Team_ sounds to me like it only includes a small set of people who regularly contribute and organized somehow as a team, so maybe committers?